Automattic/jetpack
# REST API: add functionality for saving Business Address widget in JPO (alternative)

An alternative approach to #8480  and #8426. 

#### Changes proposed in this Pull Request:
* adds functions required to inject a Business Address widget or update an existing one,
* saves Business Address data in an option,
* enables widget module if inactive ( throws an error on failure ).

#### Testing instructions:

* Follow the instructions https://github.com/Automattic/jetpack/pull/8426 tested on a site with a Business Plan ( or temporarily set `isBusiness` to `true` in main.jsx). 
* To test that it works for the initially inactive `widgets` module: 
 use `wp jetpack module deactivate` to deactivate the module and follow the previous instructions. After, verify that the module is listed as active in `wp jetpack module list` and that the widget has been set correctly. 
* Verify that the `jpo_business_address` option is set i.e. using `wp option get jpo_business_address`.
